src/applications/disability-benefits/disability-rating-calculator/components/RatingRow.jsx

```diff
@@ -1,76 +1,105 @@
-  inputRef,
-  disabled,
-}) => {
-  const onRatingChange = e => {
+class RatingRow extends React.Component {
+  onRatingChange = e => {
     const val = e.target.value;
     const re = /^[0-9\b]+$/;
 
     if (re.test(val) || val === '') {
-      updateDisability(indx, {
-        ...disability,
+      this.props.updateDisability(this.props.indx, {
+        ...this.props.disability,
         rating: val.length ? Number(val) : val,
       });
     }
   };
 
-  const onDescriptionChange = e => {
-    updateDisability(indx, { ...disability, description: e.target.value });
+  onDescriptionChange = e => {
+    this.props.updateDisability(this.props.indx, {
+      ...this.props.disability,
+      description: e.target.value,
+    });
   };
 
-  const rowId = `disability-row-${indx + 1}`;
+  render() {
+    const rowId = `disability-row-${this.props.indx + 1}`;
+    const containerClasses = classNames(
+      'vads-u-padding-x--4 vads-u-margin-y--1',
+      this.props.disability.hasError
+        ? 'vads-u-border-left--4px vads-u-border-color--secondary-dark'
+        : null,
+    );
 
-  return (
-    <div className="rating vads-l-row">
-      <span id={rowId} className="sr-only">
-        row {indx + 1}
-      </span>
-      <div className="vads-l-col--3 vads-u-padding-right--2">
-        <input
-          type="text"
-          min="0"
-          onChange={onRatingChange}
-          className="ratingInput"
-          maxLength="2"
-          value={disability.rating}
-          pattern="\d+"
-          name="rating"
-          ref={inputRef}
-          aria-labelledby={`ratingLabel ${rowId}`}
-        />
-      </div>
-      <div className="vads-l-col--6">
-        <input
-          className="descriptionInput"
-          name="description"
-          onChange={onDescriptionChange}
-          value={disability.description}
-          aria-labelledby={`descriptionLabel ${rowId}`}
-        />
-      </div>
-      <div className="vads-l-col--3">
-        <div>
-          <button
-            type="button"
-            aria-label={`Delete disability rating row ${indx + 1}`}
-            onClick={removeDisability(indx)}
-            className="va-button-link delete-btn vads-u-margin--1p5"
-            disabled={disabled}
-          >
-            <i className="fas fa-trash-alt vads-u-padding-right--0p5" />
-            Delete
-          </button>
+    const ratingInputClasses = classNames(
+      'ratingInput',
+      this.props.disability.hasError
+        ? 'vads-u-border--3px vads-u-border-color--secondary-dark'
+        : null,
+    );
+
+    return (
+      <div className={containerClasses}>
+        {this.props.disability.hasError && (
+          <>
+            <span className="usa-input-error-label">
+              Invalid disability rating
+            </span>
+            <span className="usa-input-error-message" role="alert">
+              VA disability ratings are given in 10% increments, from 0 to 100.
+            </span>
+          </>
```

Comment:
Here's where that `hasError` property (which is calculated when the parent view handles a submit) is processed. I wanted to use our standard error input classes but this use case is slightly different since there are two input fields and the error message should span across both. See [image](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19139#issuecomment-509738116)

Reply:
Here's what it looks like to use the standard `usa-input-error` class. It would have taken more work than it was worth to try override the unwanted styles -

![image](https://user-images.githubusercontent.com/1915775/61132628-c9862f80-a489-11e9-8196-b1aebc41d886.png)